node/src/actors/chain_manager/mining.rs

```diff
@@ -836,6 +953,34 @@ fn build_block(
     (block_header, txns)
 }
 
+fn build_superblock(
+    block_headers: &[BlockHeader],
+    ars_identities: &[PublicKeyHash],
```

Comment:
Maybe add a comment explaining that these are sorted by pkh.

```suggestion
    sorted_ars_identities: &[PublicKeyHash],
```

Reply:
now are sorted by pkh, but in the future it will be sorted by bls signatures, so... maybe only with the variable name could be enough